pkg/kubelet/dockertools/docker.go

```diff
@@ -40,8 +40,9 @@ import (
 )
 
 const (
-	PodInfraContainerName = leaky.PodInfraContainerName
-	DockerPrefix          = "docker://"
+	PodInfraContainerName  = leaky.PodInfraContainerName
+	DockerPrefix           = "docker://"
+	PodInfraContainerImage = "gcr.io/google_containers/pause:0.8.0"
```

Comment:
There is a flag for this, we should use it instead of this (the old version was unused I believe).